moveit_ros/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -34,41 +34,35 @@
 
 /* Author: Ioan Sucan, Jon Binney */
 
-#include <ros/ros.h>
-#include <moveit_msgs/SaveMap.h>
-#include <moveit_msgs/LoadMap.h>
+#include <rclcpp/rclcpp.hpp>
+#include <moveit_msgs/srv/save_map.hpp>
+#include <moveit_msgs/srv/load_map.hpp>
 #include <moveit/occupancy_map_monitor/occupancy_map.h>
 #include <moveit/occupancy_map_monitor/occupancy_map_monitor.h>
-#include <XmlRpcException.h>
+// #include <XmlRpcException.h>
+#include <boost/bind.hpp>
+#include <rcutils/logging_macros.h>
+#include <rcutils/time.h>
 
 namespace occupancy_map_monitor
 {
-OccupancyMapMonitor::OccupancyMapMonitor(double map_resolution)
-  : map_resolution_(map_resolution), debug_info_(false), mesh_handle_count_(0), nh_("~"), active_(false)
-{
-  initialize();
-}
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros.occupancy_map_monitor");
```

Comment:
```suggestion
static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros_occupancy_map_monitor");
```

Nit: @henningkayser Which one should we use .?

Reply:
Both are fine, I would stick with the full lib name as we decided to do so and follow this rule in other packages.
My general concern is that this won't scale with added files and also the namespaces are less explicit (i.e. for scoping moveit, moveit_ros, etc).